frontend/src/views/components/FriendApproveCard.tsx

```diff
@@ -1,15 +1,14 @@
 import { Button, Card, Typography } from "@mui/material";
 import PeopleAltIcon from "@mui/icons-material/PeopleAlt";
 import styled from "styled-components";
-import { PropsFriendApproveCard } from "../../types";
 import { useSelector } from "react-redux";
 import { selectUser } from "../../reducer/userSlice";
-import { createClient } from "@supabase/supabase-js";
+import { client } from "../../services/supabase";
+import { Friend } from "../../types";
 
-const supabase = createClient(
-  process.env.REACT_APP_SUPABASE_URL as string,
-  process.env.REACT_APP_SUPABASE_ANON_KEY as string
-);
+export interface PropsFriendApproveCard extends Friend {
```

Comment:
In my opinion, you should not use `extends`.  Like below, specifying friend is better.

```js
interface PropsFriendApproveCard {
  friend: Friend
  getUserFriendsById: () => void;
```

 Because the engineer who doesn't know this code becomes confused, I think. For example, on [L41](https://github.com/e-0922-ja/expense-tracker-app/pull/67/files#diff-561eca94f629bc9b95f2326f1d60cdbd2bd2c36f6a75c6f55ddbf7c0e349fb2bR41), could the engineers understand whether this `id` is friend id or user id at a glance?

If you define a type like the above, you can write `friend.id`. It's clear that this is a friend id.

Reply:
Changed it.